stand/libsa/in_cksum.c

```diff
@@ -53,15 +53,15 @@ int
 in_cksum(void *p, int len)
 {
 	int sum = 0, oddbyte = 0, v = 0;
-	u_char *cp = p;
+	u_char *cp = (u_char *)p;
```

Comment:
This isn't n old workaround. casting from void * to anything has been well defined for ages.